src/legacy/server/index_patterns/service/lib/resolve_time_pattern.ts

```diff
@@ -34,12 +36,16 @@ import { callIndexAliasApi } from './es_api';
  *                            and the indices that actually match the time
  *                            pattern (matches);
  */
-export async function resolveTimePattern(callCluster, timePattern) {
+export async function resolveTimePattern(callCluster: APICaller, timePattern: string) {
   const aliases = await callIndexAliasApi(callCluster, timePatternToWildcard(timePattern));
 
-  const allIndexDetails = chain(aliases)
-    .reduce((acc, index, indexName) => acc.concat(indexName, Object.keys(index.aliases || {})), [])
-    .sort()
+  const allIndexDetails = chain<string[]>(aliases as string[])
+    .reduce(
+      (acc: string[], index: any, indexName: string) =>
+        acc.concat(indexName, Object.keys(index.aliases || {})),
+      []
+    )
+    .sortBy((indexName: string) => indexName)
```

Comment:
`sort()` doesn't work here because return type of `chain` isn't array. I had to use `lodash` equivalent.

Reply:
👍 Verified they both do the same thing, apparently ts just prefers this version.